### DISCUSSION TITLE
Open project not editable by everyone

### DISCUSSION THREAD

**Discussion by sachatk** [opengisch/QField#4277] [Q&A]
I was wondering how I can set my project open to everybody so it can be edited by any user. It seems that when I set my project to "public" people can download my project, but nobody can edit it, as the pencil is not present (see picture hereunder). I always need to add the user as collaborators which is time-consuming as we want to do a mapping exercise with 300 students in Kinshasa. 

![Qfield_project](https://github.com/opengisch/QField/assets/92298302/e37c5a69-9c3e-4b61-ab73-a04c84793bff)

**Comment by ZsanettMed** [comment]
Hi Sacha, 

Private projects are accessible only by users that are added as project collaborators. Since we already discussed it via mail, I will close this ticket. 

Best regards, 
Zsanett
